opentelemetry-otlp/tests/integration_test/tests/logs.rs

```diff
@@ -12,7 +12,18 @@ use std::fs::File;
 use std::os::unix::fs::MetadataExt;
 
 fn init_logs() -> Result<sdklogs::LoggerProvider, LogError> {
-    let exporter = LogExporter::builder().with_tonic().build()?;
+    let exporter_builder = LogExporter::builder();
+    #[cfg(feature = "tonic-client")]
+    let exporter_builder = exporter_builder.with_tonic();
+    #[cfg(not(feature = "tonic-client"))]
+    #[cfg(any(
+        feature = "hyper-client",
+        feature = "reqwest-client",
+        feature = "reqwest-blocking-client"
+    ))]
```

Comment:
Would be nice to combine those two though. Maybe we can use [cfg_if](https://docs.rs/cfg-if/latest/cfg_if) it's dev only so it's easier to introduce the dependencies